### PR TITLE
Changed setup.sh

### PR DESCRIPTION
**Pull request by hpanago**
With a quick look, some changes to setup.sh for a "smoother experience".

Please check this and give me some feedback.

Thanks

(will create a new branch for more pull request, forgot, sorry)

### CODE REVIEW

**Review by kerk12 — APPROVED** [review]
It's ok for now, although, we need to implement a rollback feature that checks each step and if it has failed it will delete the successfully copied files. (so that there aren't unusable files left back). Go ahead and merge.

File: setup.sh [hunk 1]
```diff
@@ -3,15 +3,39 @@
...
+    else
+        echo "Very well. If you wish to do it later, simply run the command 'sudo update-rc.d VMInit defaults.'"
+    fi
+    #maybe there is better way for this
+    success=$(($retval1 + $retval2 + $retval3))
+
+    if [[ $success -eq 0 ]]; then
+        #some details for the user, it's always nice to know where scripts are 
+        echo "The required files have been copied successfully."
+        echo "The init script at /etc/init.d/VMInit has beenn enabled with update-rc.d VMInit defaults"
```
Comment (kerk12) [inline setup.sh]: Can you please fix the typo here?

**Review by kerk12 — CHANGES_REQUESTED** [review]

File: setup.sh (hunk 1 above)
Comment (hpanago) [reply setup.sh]: that "beenn" right? Done.

**Review by kerk12 — APPROVED** [review]

**Comment by hpanago** [comment]
"It's ok for now, although, we need to implement a rollback feature that checks each step and if it has failed it will delete the successfully copied files. (so that there aren't unusable files left back). "

Sounds perfect. Will try to add that functionality.

**Comment by kerk12** [comment]
OK, it's ready to go. Go ahead and merge.

**Comment by kerk12** [comment]
@hpanago I'll open an issue. You can have a go there.

**Pull request merged**